crates/ekore/src/anomalous_dimensions/unpolarized/spacelike/as4.rs

```diff
@@ -0,0 +1,71 @@
+//! |N3LO| |QCD|.
+
+mod ggg;
+mod ggq;
+mod gnsm;
+mod gnsp;
+mod gnsv;
+mod gps;
+mod gqg;
+
+use crate::harmonics::cache::Cache;
+use num::complex::Complex;
+
+pub use ggg::gamma_gg;
+pub use ggq::gamma_gq;
+pub use gnsm::gamma_nsm;
+pub use gnsp::gamma_nsp;
+pub use gnsv::gamma_nsv;
+pub use gps::gamma_ps;
+pub use gqg::gamma_qg;
+
+/// Compute the singlet anomalous dimension matrix.
+///
+/// `variation = (gg, gq, qg, qq)` is a list indicating which variation should
+/// be used. `variation = 1,2` is the upper/lower bound, while any other value
+/// returns the central (averaged) value.
```

Comment:
Let's lift that from here to the top level, i.e. `ad::u::s::gamma_singlet_qcd` and add an explicit reference to that instead (something like "for the sorting of variation see blub")

Reply:
done in 5882487e